diff --git a/Makefile b/Makefile
index c3ba9be..868a7e4 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,13 @@
+KERNELRELEASE ?= $(shell uname -r)
+KERNEL_DIR  ?= /lib/modules/$(KERNELRELEASE)/build
 obj-m += acer-wmi-battery.o
 PWD := $(CURDIR)
 
 all:
-	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) modules
+	make -C $(KERNEL_DIR) M=$(PWD) modules
+
+install:
+	make -C $(KERNEL_DIR) M=$(PWD) modules_install
 
 clean:
-	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) clean
+	make -C $(KERNEL_DIR) M=$(PWD) clean
